webpack.config.common.js

```diff
@@ -50,6 +50,8 @@ module.exports = {
       'react/jsx-dev-runtime.js': 'react/jsx-dev-runtime',
       'react/jsx-runtime.js': 'react/jsx-runtime',
       src: path.resolve(__dirname, 'src/'),
+      components: path.resolve(__dirname, 'src', 'components'),
```

Comment:
can we setup to use anything from `src`? not specify

Reply:
ok, added src as alias too